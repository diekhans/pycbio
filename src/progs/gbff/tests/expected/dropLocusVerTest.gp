XM_379029.2	NT_011516	-	105517	108827	107305	107938	1	105517,	108827,
XM_379030.2	NT_011516	-	153532	192484	190984	191416	6	153532,162182,173509,186527,190466,192187,	154015,162444,173575,189938,191774,192484,
XM_497788.1	NT_011516	+	113129	122124	113129	122124	3	113129,121459,121951,	113180,121607,122124,
XM_497790.1	NT_011519	+	153654	208023	153654	208023	6	153654,205067,206164,206463,207504,208015,	153756,205191,206253,206567,207680,208023,
NM_014570.3	NT_011520	-	22583044	22643813	22643813	22643813	16	22583044,22585557,22593591,22595285,22597331,22603660,22604247,22608788,22610174,22613458,22618067,22620760,22621880,22627462,22634032,22643632,	22584124,22585679,22593682,22595409,22597463,22603783,22604376,22608928,22610221,22613518,22618155,22620844,22622012,22627535,22634151,22643813,
NM_004327.2	NT_011520	+	2913268	3048713	2913716	3048278	23	2913268,2986554,2993705,2994110,3001163,3004287,3005836,3006389,3016732,3017788,3019914,3020852,3022272,3023094,3025296,3027779,3042179,3043079,3044452,3045642,3046723,3047307,3048188,	2914995,2986736,2993810,2994296,3001271,3004348,3005889,3006530,3016854,3017957,3020034,3020928,3022377,3023169,3025394,3027911,3042239,3043189,3044592,3045777,3046829,3047470,3048713,
NM_021574.1	NT_011520	+	2913120	3048706	2913716	3048278	22	2913120,2986554,2993705,2994110,3001163,3004287,3005836,3006389,3016732,3017788,3019914,3020852,3022272,3023094,3025296,3042179,3043079,3044452,3045642,3046723,3047307,3048188,	2914995,2986736,2993810,2994296,3001271,3004348,3005889,3006530,3016854,3017957,3020034,3020928,3022377,3023169,3025394,3042239,3043189,3044592,3045777,3046829,3047470,3048706,
NM_001585.2	NT_011520	+	23211426	23261354	23261354	23261354	4	23211426,23221466,23260961,23261141,	23211728,23221648,23260974,23261354,
NM_005207.2	NT_011520	+	662282	698459	662791	694702	3	662282,678635,694567,	663102,679101,698459,
XM_496513.1	NT_011520	-	3050284	3051512	3050744	3051421	2	3050284,3051229,	3050858,3051512,
NM_032262.1	NT_011520	+	445970	449460	446041	446524	8	445970,447442,447862,448324,448489,448824,449044,449167,	447163,447566,447979,448381,448549,448897,449085,449460,
NM_144704.1	NT_011520	+	710002	726218	712801	725889	21	710002,712661,718164,718615,718920,719109,719382,719561,720549,721072,721265,721508,721724,721889,722557,722745,724128,724484,724877,725604,725849,	710064,712832,718378,718725,719030,719154,719479,719674,720636,721164,721396,721624,721796,721953,722634,722863,724227,724560,724982,725625,726218,
NM_152509.1	NT_011520	-	3123361	3135311	3132559	3133063	3	3123361,3132515,3135116,	3124480,3133339,3135311,
NM_152612.2	NT_011520	+	1377654	1382185	1378039	1381928	5	1377654,1377977,1378879,1379542,1381289,	1377685,1378111,1379428,1380124,1382185,
NM_207474.1	NT_011520	+	847873	866642	862946	865823	6	847873,860786,861981,862898,863644,865661,	848084,860859,862116,863167,863807,866642,
NM_080839.4	NT_011520	+	2379350	2380940	2379384	2380780	5	2379350,2379792,2380021,2380215,2380633,	2379560,2379920,2380134,2380329,2380940,
NM_199127.1	NT_011520	+	2379348	2380940	2379384	2380780	4	2379348,2379792,2380021,2380633,	2379560,2379920,2380329,2380940,
NM_002073.2	NT_011520	+	2803237	2857793	2828451	2856187	3	2803237,2828002,2855842,	2803325,2829174,2857793,
NM_015094.1	NT_011520	+	1162261	1196317	1187689	1191601	3	1162261,1187616,1189779,	1162418,1187715,1196317,
NM_020070.2	NT_011520	-	3305880	3313064	3306021	3312946	3	3305880,3307722,3312740,	3306341,3307838,3313064,
NM_152855.1	NT_011520	-	3305880	3313064	3306292	3312946	2	3305880,3312740,	3306341,3313064,
NM_080842.1	NT_011520	+	2385249	2387119	2385249	2387119	2	2385249,2386931,	2385400,2387119,
XM_086876.6	NT_011520	-	412451	416042	412674	416042	2	412451,415321,	413081,416042,
XM_036942.3	NT_011520	-	1290214	1295834	1290914	1295834	1	1290214,	1295834,
XM_097886.7	NT_011520	-	1372949	1374909	1373275	1374872	5	1372949,1373867,1374165,1374459,1374708,	1373645,1374045,1374265,1374619,1374909,
NM_131915.2	NT_011520	-	2365114	2366556	2365308	2365617	2	2365114,2366512,	2365620,2366556,
NM_201565.1	NT_011520	+	927771	937013	928134	929238	4	927771,928517,930978,936579,	928273,930445,931935,937013,
XM_497792.1	NT_011520	+	377350	399696	377350	399696	3	377350,377446,397757,	377355,377620,399696,
XM_371423.2	NT_011520	+	1027311	1029715	1027738	1028281	1	1027311,	1029715,
XM_371430.2	NT_011520	+	1114830	1118664	1114830	1118664	5	1114830,1115369,1116006,1116651,1118627,	1115135,1115518,1116170,1116758,1118664,
XM_371455.3	NT_011520	-	3200525	3219736	3202238	3219662	11	3200525,3202092,3202751,3203070,3203563,3206133,3206632,3213172,3215330,3215823,3219634,	3200574,3202243,3202793,3203092,3203919,3206167,3206694,3213256,3215361,3215882,3219736,
XM_372889.2	NT_011520	+	1086710	1104642	1086710	1104642	13	1086710,1087023,1087973,1088367,1088959,1089808,1090071,1101294,1101679,1103114,1103334,1103607,1104547,	1086949,1087893,1088249,1088619,1089609,1089916,1090192,1101443,1101904,1103225,1103492,1103671,1104642,
XM_372891.2	NT_011520	-	1053450	1086605	1053450	1086605	9	1053450,1056546,1057096,1059157,1064127,1066052,1067653,1085792,1086517,	1054382,1056631,1057158,1059223,1064214,1066119,1067712,1085942,1086605,
XM_497819.1	NT_011520	-	22574457	22581496	22574457	22581496	7	22574457,22578328,22579531,22579956,22580731,22580934,22581446,	22574512,22579455,22579657,22580121,22580832,22581035,22581496,
XM_375955.2	NT_011520	-	973640	1024450	973640	1024432	5	973640,1000021,1004315,1022564,1024274,	973996,1000178,1004409,1022753,1024450,
XM_379036.2	NT_011520	+	701225	709536	701225	709376	3	701225,701914,709247,	701297,702343,709536,
XM_375963.2	NT_011520	+	790844	798337	790844	798337	6	790844,792213,792741,793808,797106,798217,	790983,792356,792874,794015,797216,798337,
XM_497793.1	NT_011520	+	799974	825874	799974	825874	14	799974,801962,805434,810319,811808,812317,812710,814658,815916,819235,820355,822058,822968,825716,	800034,802058,805520,810579,811950,812496,812859,814904,816116,819289,820416,822178,823098,825874,
XM_497817.1	NT_011520	+	14433217	14837184	14433217	14837184	14	14433217,14434420,14438388,14439080,14484110,14484503,14485083,14488896,14807573,14810624,14819874,14835302,14836007,14836312,	14433650,14434593,14438444,14439219,14484178,14484653,14485196,14488988,14807798,14810870,14820070,14835920,14836220,14837184,
XM_498459.1	NT_011520	+	747077	754746	747307	753528	2	747077,753524,	747687,754746,
XM_496500.1	NT_011520	+	446863	449460	446863	449347	8	446863,447308,447901,448324,448489,448824,449044,449167,	447241,447566,447979,448381,448608,448913,449085,449460,
XM_496501.1	NT_011520	-	777597	789897	777597	789812	4	777597,787200,789006,789741,	777642,787272,789157,789897,
XM_496502.1	NT_011520	-	870517	872921	871105	872494	1	870517,	872921,
XM_496503.1	NT_011520	-	946604	952288	946604	952288	6	946604,948312,949608,950251,951452,951955,	946793,948493,949717,950358,951556,952288,
XM_290331.5	NT_011520	-	952832	972023	952995	970412	15	952832,953446,953641,953856,954216,957922,958582,960782,961328,966723,966943,970248,970962,971157,971451,	953142,953560,953754,953984,954404,958059,958732,960936,961525,966812,967072,970419,971039,971244,972023,
XM_498871.1	NT_011520	-	1039892	1040803	1040060	1040342	2	1039892,1040787,	1040485,1040803,
XM_036936.3	NT_011520	+	1128716	1134336	1128716	1133636	1	1128716,	1134336,
XM_496504.1	NT_011520	+	1401642	1407221	1401752	1405831	6	1401642,1402638,1403123,1403500,1405694,1407110,	1401819,1402916,1403320,1403544,1405879,1407221,
XM_498872.1	NT_011520	+	1501792	1507731	1501792	1507191	3	1501792,1506865,1507473,	1501919,1507419,1507731,
XM_498875.1	NT_011520	+	3126268	3127548	3126361	3127548	6	3126268,3126533,3126697,3126935,3127246,3127370,	3126451,3126593,3126762,3126993,3127296,3127548,
XM_496515.1	NT_011520	+	3272899	3274449	3273003	3273501	1	3272899,	3274449,
XM_498876.1	NT_011520	-	3313636	3316615	3315926	3316211	3	3313636,3315888,3316564,	3314319,3316234,3316615,
NM_080924.1	NT_011520	-	1862442	1862943	1862442	1862943	1	1862442,	1862943,
NM_080926.1	NT_011520	+	2048157	2053657	2052039	2053657	6	2048157,2048876,2051766,2052039,2052468,2053630,	2048231,2049048,2051955,2052222,2052525,2053657,
NM_006767.2	NT_011520	+	727224	743895	727229	742206	21	727224,727884,730698,732361,732867,733646,734482,735243,736485,737071,737651,738519,738781,738961,739415,739727,740603,740820,741553,741743,742089,	727429,727947,730755,732441,732976,733730,734540,735383,736687,737227,737762,738612,738877,739127,739585,739884,740730,740970,741659,741824,743895,
NM_002745.2	NT_011520	-	1507447	1612539	1514061	1612299	10	1507447,1508152,1514052,1517730,1533114,1533551,1543869,1550707,1552521,1612180,	1508001,1509098,1514178,1517840,1533246,1533666,1543986,1550897,1552704,1612539,
NM_138957.1	NT_011520	-	1513900	1612539	1514061	1612299	8	1513900,1517730,1533114,1533551,1543869,1550707,1552521,1612180,	1514178,1517840,1533246,1533666,1543986,1550897,1552704,1612539,
NM_145042.2	NT_011520	-	753066	759003	753963	756362	3	753066,756241,758639,	754307,756448,759003,
NM_005446.2	NT_011520	+	760017	773688	760062	771475	12	760017,760799,762846,767533,767799,767975,768132,768366,770662,770868,771109,771277,	760196,760950,762918,767609,767893,768056,768274,768476,770756,770934,771187,773688,
NM_002650.1	NT_011520	-	452559	479524	452903	479407	23	452559,454152,454764,455555,456197,457342,457549,458137,459314,459484,462564,463549,466154,471344,472086,472630,474169,474488,474748,477827,478592,478899,479251,	452955,454236,454854,455715,456318,457468,457654,458247,459385,459557,462635,463679,466275,471398,472254,472721,474363,474568,474868,477955,478700,479051,479524,
NM_058004.1	NT_011520	-	452559	583570	452903	583570	54	452559,454152,454764,455555,456197,457342,457549,458137,459314,459484,462564,463549,466154,471344,472086,472630,474169,474488,474748,477827,478592,478899,479251,487084,487464,489484,492439,494761,496149,496530,497759,497966,506160,509673,509956,510441,538049,541002,543440,543949,544527,546829,548047,549156,549830,552218,555823,558214,563339,564497,565395,569182,579418,583471,	452955,454236,454854,455715,456318,457468,457654,458247,459385,459557,462635,463679,466275,471398,472254,472721,474363,474568,474868,477955,478700,479051,479410,487212,487621,489577,492559,494851,496233,496612,497884,498037,506243,509807,510093,510550,538100,541171,543544,544133,544623,546962,548177,549257,550022,552315,555889,558363,563406,564757,565468,569271,579512,583570,
NM_014348.1	NT_011520	-	2375838	2377125	2375838	2377125	1	2375838,	2377125,
NM_014337.2	NT_011520	+	1410844	1441281	1410957	1440348	20	1410844,1414770,1415423,1415854,1417187,1419921,1426156,1427294,1428039,1429610,1431340,1431749,1432500,1432930,1433589,1438673,1439450,1439612,1439794,1440251,	1410989,1414820,1415469,1415917,1417239,1419973,1426248,1427384,1428115,1429771,1431416,1431856,1432590,1432964,1433707,1438730,1439523,1439675,1439928,1441281,
NM_148175.1	NT_011520	+	1410879	1442729	1410957	1440348	21	1410879,1414770,1415423,1415854,1417187,1419921,1426156,1427294,1428039,1429610,1431340,1431749,1432500,1432930,1433589,1438673,1439450,1439612,1439794,1440251,1441604,	1410989,1414820,1415469,1415917,1417239,1419973,1426248,1427384,1428115,1429771,1431416,1431856,1432590,1432964,1433707,1438730,1439523,1439675,1439928,1440354,1442729,
NM_148176.1	NT_011520	+	1410878	1442729	1410957	1441761	21	1410878,1414770,1415423,1415854,1417187,1419921,1426156,1427294,1428039,1429610,1431340,1431749,1432500,1432930,1433589,1438673,1439450,1439612,1439794,1440255,1441742,	1410989,1414820,1415469,1415917,1417239,1419973,1426248,1427384,1428115,1429771,1431416,1431856,1432590,1432964,1433707,1438730,1439523,1439675,1439928,1440354,1442729,
NM_014634.2	NT_011520	-	1664361	1697786	1668033	1690989	8	1664361,1670510,1676088,1678331,1678964,1684472,1690783,1697732,	1668413,1670604,1676232,1678520,1679167,1684621,1691049,1697786,
NM_006115.2	NT_011520	-	2280691	2292251	2281057	2289821	6	2280691,2282716,2283757,2289800,2290533,2292150,	2281634,2283325,2284080,2289919,2290569,2292251,
NM_004914.2	NT_011520	+	2878081	2897100	2878121	2894320	11	2878081,2879360,2882818,2885216,2885788,2888723,2890747,2891653,2891918,2893634,2894255,	2878307,2879441,2882910,2885282,2885890,2888788,2890799,2891735,2892009,2893754,2897100,
NM_014433.2	NT_011520	-	2792162	2874810	2792208	2873176	7	2792162,2794555,2796648,2866781,2871595,2872977,2874704,	2792465,2794692,2796880,2866900,2871698,2873228,2874810,
NM_022044.1	NT_011520	+	1387174	1389157	1387194	1389033	3	1387174,1387719,1388751,	1387381,1387916,1389157,
NM_000185.2	NT_011520	+	518955	532577	524169	531923	5	518955,524153,528828,530860,531731,	519002,525058,529102,531005,532577,
NM_004173.1	NT_011520	-	773576	776710	773912	776670	4	773576,774193,774568,775688,	774088,774302,775209,776710,
NM_004782.2	NT_011520	+	603860	634678	603967	632693	5	603860,615193,625905,628327,632535,	604204,615390,625991,628426,634678,
NM_080740.3	NT_011520	-	2258628	2265182	2258894	2260523	2	2258628,2265000,	2260594,2265182,
NM_080764.2	NT_011520	-	2229335	2254074	2232660	2234292	4	2229335,2238950,2252696,2253545,	2234360,2239068,2252757,2254074,
NM_030573.1	NT_011520	-	744629	746944	744737	746769	4	744629,745504,746113,746689,	745290,745645,746269,746944,
NM_003935.3	NT_011520	-	1701972	1727716	1702054	1720650	18	1701972,1703432,1704072,1704527,1705261,1707369,1707687,1708863,1709101,1710225,1712543,1713559,1715150,1716817,1717552,1719297,1720580,1727379,	1702536,1703634,1704173,1704677,1705390,1707543,1707834,1708969,1709256,1710316,1712657,1713716,1715347,1716892,1717659,1719429,1720748,1727716,
NM_003347.2	NT_011520	+	1312525	1368892	1312602	1366527	4	1312525,1337718,1355714,1366372,	1312629,1337814,1355901,1368892,
NM_198157.1	NT_011520	+	1312525	1368892	1312602	1359470	5	1312525,1337718,1355714,1359414,1366372,	1312629,1337814,1355901,1359523,1368892,
NM_007128.2	NT_011520	+	1989768	1990495	1989794	1990318	2	1989768,1989926,	1989840,1990495,
NM_013313.3	NT_011520	-	1442396	1480640	1445986	1455602	5	1442396,1448227,1448696,1455485,1480472,	1446076,1448336,1448740,1455766,1480640,
NM_033226.1	NT_010498	-	1731082	1794880	1731824	1794534	29	1731082,1732018,1733702,1734850,1736035,1736654,1739229,1744855,1748981,1752251,1753248,1755431,1756549,1759545,1759687,1759885,1763524,1765370,1769825,1772322,1776568,1778897,1781796,1786337,1787272,1788796,1789311,1792019,1794415,	1731911,1732183,1733816,1734929,1736195,1736844,1739319,1745012,1749119,1752478,1753446,1755535,1756639,1759617,1759772,1760020,1763728,1765443,1769950,1772394,1776847,1779005,1781945,1786485,1787446,1789026,1789463,1792175,1794880,
NM_145187.1	NT_010498	-	1731082	1794880	1745041	1794534	28	1731082,1732018,1733702,1736049,1736654,1739229,1744855,1748981,1752251,1753248,1755431,1756549,1759545,1759696,1759885,1763524,1765370,1769825,1772322,1776568,1778897,1781796,1786337,1787272,1788796,1789315,1792019,1794415,	1731911,1732183,1733816,1736195,1736844,1739319,1745042,1749119,1752478,1753446,1755535,1756639,1759617,1759772,1760020,1763728,1765443,1769950,1772394,1776847,1779005,1781945,1786485,1787446,1789030,1789463,1792175,1794880,
NM_145188.1	NT_010498	-	1731082	1794880	1745036	1794534	28	1731082,1732018,1733702,1734850,1736049,1736654,1739229,1744855,1748981,1753248,1755431,1756549,1759545,1759696,1759885,1763524,1765370,1769825,1772322,1776568,1778897,1781796,1786337,1787272,1788796,1789315,1792019,1794415,	1731911,1732183,1733816,1734929,1736195,1736844,1739319,1745042,1749119,1753446,1755535,1756639,1759617,1759772,1760020,1763728,1765443,1769950,1772394,1776847,1779005,1781945,1786485,1787446,1789030,1789463,1792175,1794880,
NM_145189.1	NT_010498	-	1731082	1794880	1745041	1794534	29	1731082,1732018,1733702,1734850,1736035,1736654,1739229,1744855,1748981,1752251,1753248,1755431,1756549,1759545,1759696,1759885,1763524,1765370,1769825,1772322,1776568,1778897,1781796,1786337,1787272,1788796,1789315,1792019,1794415,	1731911,1732183,1733816,1734929,1736195,1736844,1739319,1745042,1749119,1752478,1753446,1755535,1756639,1759617,1759772,1760020,1763728,1765443,1769950,1772394,1776847,1779005,1781945,1786485,1787446,1789030,1789463,1792175,1794880,
NM_145190.1	NT_010498	-	1731082	1794880	1745041	1794534	28	1731082,1732018,1733702,1736035,1736654,1739229,1744855,1748981,1752251,1753248,1755431,1756549,1759545,1759696,1759885,1763524,1765370,1769825,1772322,1776568,1778897,1781796,1786337,1787272,1788796,1789315,1792019,1794415,	1731911,1732183,1733816,1736195,1736844,1739319,1745042,1749119,1752478,1753446,1755535,1756639,1759617,1759772,1760020,1763728,1765443,1769950,1772394,1776847,1779005,1781945,1786485,1787446,1789030,1789463,1792175,1794880,
NM_004876.1	NT_011295	+	15463617	15504824	15463751	15504428	5	15463617,15482383,15482991,15502697,15504166,	15463781,15482510,15483087,15503242,15504824,
NM_004166.3	NT_010799	-	9047671	9065992	9047816	9050665	7	9047671,9048353,9050586,9061736,9062295,9062881,9065435,	9047904,9048468,9050720,9061876,9062407,9062941,9065992,
NM_032962.2	NT_010799	-	9047671	9065992	9047816	9050665	8	9047671,9048353,9049763,9050586,9061736,9062295,9062881,9065435,	9047904,9048468,9049811,9050720,9061876,9062407,9062941,9065992,
NM_032963.2	NT_010799	-	9047671	9050719	9047816	9050665	3	9047671,9048353,9050586,	9047904,9048468,9050719,
NM_004167.3	NT_010799	-	9047671	9065992	9061782	9065511	8	9047671,9048353,9049763,9050586,9061736,9062295,9062881,9065435,	9047904,9048468,9049811,9050720,9061876,9062407,9062941,9065992,
NM_032964.2	NT_010799	-	9047671	9065992	9061782	9065511	7	9047671,9048353,9050586,9061736,9062295,9062881,9065435,	9047904,9048468,9050720,9061876,9062407,9062941,9065992,
NM_032965.2	NT_010799	-	9061643	9065992	9061782	9065511	4	9061643,9062295,9062881,9065435,	9061876,9062407,9062941,9065992,
NM_033108.2	NT_010799	+	6859383	6861288	6859483	6861167	2	6859383,6860474,	6859996,6861288,
NM_152584.1	NT_010799	+	6859386	6901657	6859483	6901330	2	6859386,6901231,	6859996,6901657,
NR_003510.1	NT_010799	+	6859386	6901659	6901659	6901659	6	6859386,6862809,6863478,6864612,6895368,6901231,	6859996,6862919,6863565,6864767,6895454,6901659,
