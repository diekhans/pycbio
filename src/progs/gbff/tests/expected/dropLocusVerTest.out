Warning: can't find entry for CDS NP_975011.1 in input/hs_chr22.part.q
